packages/web/docs/src/components/landing-page-feature-tabs.tsx

```
@@ -0,0 +1,150 @@
+      image: auditImage,
+      // TODO: image
+    },
+  ],
+};
+
+export interface LandingPageFeatureTabsProps {
+  className?: string;
+}
+
+export function LandingPageFeatureTabs({ className }: LandingPageFeatureTabsProps) {
+  const icons = [<SchemaRegistryIcon />, <GraphQLObservabilityIcon />, <GatewayIcon />];
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)